### PR TITLE
Bug/#70/bracket calculation

### LINKED ISSUE #70: Correct calculations of brackets

### PR DESCRIPTION
**Pull request by alfetopito**
# Description

Closes #70 

Added proper bracket calculation, same as used on `dex-liquidity` scripts

- To calculate the brackets, is required to have all prices and total brackets inputs filled

screenshot_2020-09-25_14-47-20

# To Test
- [ ] Open CMM app
- [ ] Play around with prices and brackets
- [ ] Deploy strategy and verify brackets funding match displayed values

# Background

N/A

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -131,7 +133,7 @@ module.exports = {
   // snapshotSerializers: [],
 
   // The test environment that will be used for testing
-  testEnvironment: "node",
+  testEnvironment: "jsdom",
```
Comment (bh2smith) [inline jest.config.js:136]: not sure if this is a typo meant to be "json", or if actually intended to be "jsdom"

File: test/globalSetup.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module.exports = (): void => {
+  process.env.NETWORK = "rinkeby";
+  process.env.INFURA_API_KEY = "213kldjalpi";
```
Comment (bh2smith) [inline test/globalSetup.ts:3]: I guess we don't mind committing this?

**Review by bh2smith — APPROVED** [review]
Cool tests! Looks good, but I must admit that some of this front-end stuff is a bit of a mystery to me!

File: jest.config.js (hunk 1 above)
Comment (andre-meyer) [reply jest.config.js:136]: jsdom means js with dom simulation :)

**Review by andre-meyer — APPROVED** [review]

File: test/globalSetup.ts (hunk 2 above)
Comment (alfetopito) [reply test/globalSetup.ts:3]: Of course not a real API key.
Those env vars are required to be set, but I don't want to have to set them when running the tests.
So, anything will do.

**Comment by elena-zh** [comment]
Verified. Works fine. 
Number of brackets and funding amounts/transfers are correct after a strategy deployment.

**Pull request merged**